blocksuite/affine/data-view/src/property-presets/progress/define.ts

```diff
@@ -21,4 +21,5 @@ export const progressPropertyModelConfig =
       return value;
     },
     isEmpty: () => false,
+    defaultValue: 0,
```

Comment:
The progress value always ranges from 0 to 100. When a new progress item is created, it is initially set to 0.